src/js/sdk.ts

```diff
@@ -118,3 +118,18 @@ export function nativeCrash(): void {
     client.nativeCrash();
   }
 }
+
+/**
+ * Closes the SDK, stops sending events.
+ */
+export async function close(): Promise<void> {
+  try {
+    const client = getCurrentHub().getClient<ReactNativeClient>();
+
+    if (client) {
+      await client.close();
+    }
+  } catch (e) {
+    throw new SentryError("Failed to close the SDK");
```

Comment:
We shouldn't throw here though. Lets just log with Error level but we shouldn't be potentially crashing the app because the SDK failed to close